features/step_definitions/cucumber/selenium.js

```
@@ -5,6 +5,21 @@ const { suite } = require('selenium-webdriver/testing');
 const { expect } = require('chai');
 require("chromedriver");
 
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)